.settings/org.eclipse.core.resources.prefs

```diff
@@ -133,6 +133,8 @@ encoding//Debug/reliance_edge/tools/subdir_rules.mk=UTF-8
 encoding//Debug/reliance_edge/tools/subdir_vars.mk=UTF-8
 encoding//Debug/reliance_edge/util/subdir_rules.mk=UTF-8
 encoding//Debug/reliance_edge/util/subdir_vars.mk=UTF-8
+encoding//Debug/source/athena_drivers/subdir_rules.mk=UTF-8

```

Comment:
Does this still exist?

Reply:
I'm not sure of the purpose of this file to be able to tell you

Reply:
I think it has to do sth with includes. And, I don't think we have anything such as athena_drivers anymore. Do we?
It's probably because you had this local repo before the submodule refactor. And somehow pulling from the upstream hasn't overwritten it. Not 100% sure and probably not a big deal.